src/backend/marsha/core/views.py

```diff
@@ -108,7 +109,7 @@ def _get_app_data(self):
                 resource=lti.resource_id,
             )
 
-            app_data = cache.get(cache_key)
+            app_data, origin_context_id = cache.get(cache_key, [None, None])
```

Comment:
You're making things more complex when in fact the origin context id should be a property of the resource.

In `app_data`, resource is computed as:
```python
"resource": self.serializer_class(resource).data if resource else None
```

Maybe it could be a property on the serialiser?

Reply:
we should introduce a new object in the serialized resource ? 
```
{
  playlist: {
    lti_id: "foo",
  },
}
```

Reply:
either this or "origin_context_id" like you started... I have no opinion on which one is the best :man_shrugging:

Reply:
We can maybe create a `PlaylistSerializer` and add it as a side resource like we do for `thumbnail`. The day we want to expose the `Playlist` resource the Serializer will already exists.

Reply:
:+1:

Reply:
and for now we can expose `title` and `lti_id`

Reply:
But it will be now more complex in the front application :smile: We have to refactor the instructor view, all this components are resource agnostic.

Reply:
so maybe the `origin_context_id` field was a better idea?

Reply:
it changes nothing because in both cases we add a new property on the resource and in the front we have to use the resource.

Reply:
should we also expose the playlist id ?

Reply:
up to you :man_shrugging: